projects/frontend/data-pipelines/gui/proxy.config.json

```diff
@@ -1,6 +1,6 @@
 {
   "/data-jobs": {
-    "target": "http://cicd-control-service-svc:8092",
+    "target": "https://iaclqhm5xk.execute-api.us-west-1.amazonaws.com",
```

Comment:
I found in the VDK documentation that this is the public API that could be used for testing purposes and I guess it would be better proxy.config.json to be pointing directly there instead of making port-forwarding in k8s.
I wasn't able to find kubeconfig for VDK environment in the place where you are storing secrets, maybe I don't have permission for that directory :/

Reply:
That is indeed the public API that could be used for testing . And it's better to use it then port forwarding. 

For Access to Gitlab CI (https://gitlab.com/vmware-analytics/versatile-data-kit) you need to have gitlab user id and I can add you to the group there.

Reply:
Are we ok with leaking internals like that? What if we get DDOSed?

Reply:
> That is indeed the public API that could be used for testing . And it's better to use it then port forwarding.
> 
> For Access to Gitlab CI (https://gitlab.com/vmware-analytics/versatile-data-kit) you need to have gitlab user id and I can add you to the group there.

I have account in gitlab but If you could add me to the group it would be great. Thank you.